packages/js/src/ui/config/defaultVariables.ts

```diff
@@ -14,4 +14,5 @@ export const defaultVariables: Required<Variables> = {
   colorShadow: 'rgb(0,0,0)',
   fontSize: '1rem',
   borderRadius: '0.375rem',
+  colorStripes: '#FF9A68',
```

Comment:
Unfortunately, I had to add this color to the theme as it's a bit different color for the white and dark themes. I wasn't able to make it look good with just one color because we don't know if it's a dark/light theme.